scripts/deploy.py

```diff
@@ -268,19 +305,53 @@ def _execute_deployment():
 
         print 'Returning to oppia/ root directory.'
 
+    # This is required to compose the release_version_library_url correctly.
+    if '.' in current_release_version:
```

Comment:
We should probably do this at or near the very beginning, not after half the app has been deployed...

Reply:
Done.